README.md

```diff
@@ -64,6 +64,11 @@ other targets are only tested on Rust nightly.
 | `x86_64-unknown-linux-gnu` (tier 1) | ✓         | ✓       | ✓            |
 | **MacOSX targets:**                 | **build** | **run** | **jemalloc** |
 | `x86_64-apple-darwin` (tier 1)      | ✓         | ✓       | ✗            |
+| **Windows targets:**                | **build** | **run** | **jemalloc** |
```

Comment:
Can you also update the github action to include tests on Windows?

Reply:
Ok, I will try to add some test and ci for win gnu bankend.